docs/counter_example.md

```diff
@@ -1,53 +1,102 @@
-# ABCI Counter with AEAs
+# Counter Agent Service Demo
```

Comment:
I don't think this is a service. It is just a degenerate version of a service. So I would not use the term agent service here

Reply:
I was wrong.., there is a service for this ...interesting

Reply:
It seems that it is a non-FSM based service.